optaplanner-quickstarts/quarkus-facility-location/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>optaplanner-quarkus-facility-location-quickstart</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+
+  <properties>
+    <version.org.optaplanner>8.0.0-SNAPSHOT</version.org.optaplanner>
+    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
+
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <failsafe-plugin.version>2.22.1</failsafe-plugin.version>
```

Comment:
+1 The failsafe plugin must use the same version as the surefire plugin

soft suggestion: define surefire before failsafe, because surefire runs before failsafe

Reply:
Fixed. I needed to update Maven repo index in my IDE.